### ISSUE TITLE
FORMAT/RD missing from header

### ISSUE THREAD

**Issue by fgvieira** [SciLifeLab/TIDDIT#91]
I recently used `tiddit` but `FORMAT/RD` was missing from the header:
```
[...]
##FILTER=<ID=Density,Description="The discordant reads cluster too tightly">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##FORMAT=<ID=CN,Number=1,Type=Integer,Description="Copy number genotype for imprecise events">
##FORMAT=<ID=DV,Number=1,Type=Integer,Description="Number of paired-ends that support the event">
##FORMAT=<ID=RV,Number=1,Type=Integer,Description="Number of split reads that support the event">
##FORMAT=<ID=DR,Number=2,Type=Integer,Description="Number of paired-ends that supporting the reference allele (breakpoint A, and B)">
##FORMAT=<ID=RR,Number=2,Type=Integer,Description="Number of reads supporting the reference allele (breakpoint A, and B)">
##FORMAT=<ID=COV,Number=3,Type=Float,Description="Coverage (at A,B, and between)">
##FORMAT=<ID=LQ,Number=2,Type=Float,Description="Fraction of low quality reads">
##LibraryStats=TIDDIT-3.3.0 Coverage=35.559999987483025  ReadLength=150.7220850111166 MeanInsertSize=424.15720315371186 STDInsertSize=794.6088180408016 Reverse_Forward=False
[...]
```

**Comment by fgvieira** [comment]
Update, it seems it might just be a typo, since `FORMAT/DR` is in the header:
```
##FORMAT=<ID=DR,Number=2,Type=Integer,Description="Number of paired-ends that supporting the reference allele (breakpoint A, and B)">
```

**Comment by J35P312** [comment]
Thanks! I have merged it! 3.3.1 should pop up soon on bioconda!
